src/utils/env.ts

```diff
@@ -1,6 +1,8 @@
 import { existsSync, statSync } from 'fs';
 import { join } from 'path';
-const os = require('os');
+// XXX(BYK): This is to be able to spy on `homedir()` in tests
+// TODO(BYK): Convert this to ES6 imports
```

Comment:
Why is this hard?

Reply:
Eh, multiple layers of issues. Will explain later.